bats/examples.bats

```diff
@@ -23,18 +32,18 @@ teardown_file() {
       }
     }'
   )
-
   exec_graphql 'cala-outbox-import-job-create' "$variables"
-
   name=$(graphql_output '.data.calaOutboxImportJobCreate.job.name')
-  [[ "$name" == "rust-example" ]] || exit 1;
+  error_msg=$(graphql_output '.errors[0].message')
+  [[ "$name" == "rust-example" || "$error_msg" =~ duplicate.*jobs_name_key ]] || exit 1;
 
-  background cargo run --bin cala-ledger-example-rust > .rust-example-logs
+  background cargo run --bin cala-ledger-example-rust > .rust-example-logs 2>&1
 
-  sleep 5
+  job_count=$(cat .e2e-logs | grep 'Executing CalaOutboxImportJob importing' | wc -l)
+  retry 20 1 wait_for_new_import_job $job_count || true
+  sleep 1
 
   exec_graphql 'list-accounts'
-
-  name=$(graphql_output '.data.accounts.nodes[0].name')
-  [[ "$name" == "MY ACCOUNT" ]] || exit 1;
+  accounts_after=$(graphql_output '.data.accounts.nodes | length')
```

Comment:
Probably should change this assert to fetch account by name and assert by the jittered name instead of listing accounts.

Reply:
Yup, interim solution as I wasn't sure if we should implement any new queries or mutations with the tests being the only forcing function. But maybe we're early enough that this is ok to drive the api design?